src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3358,7 +3358,33 @@ private void getEEInfo(ref CORINFO_EE_INFO pEEInfoOut)
 
         private void getAsyncInfo(ref CORINFO_ASYNC_INFO pAsyncInfoOut)
         {
-            throw new NotImplementedException();
+            DefType continuation = MethodBeingCompiled.Context.SystemModule.GetType("System.Runtime.CompilerServices"u8, "Continuation"u8);
```

Comment:
(Same comment for all `GetType` uses below.)

* `GetType` can return null and we'd nullref, `GetKnownType` won't return null and will print a better error message if e.g. CoreLib is mismatched.
* Nit: context is available through a non-virtual path on `_compilation.TypeSystemContext`.

```suggestion
            DefType continuation = _compilation.TypeSystemContext.SystemModule.GetKnownType("System.Runtime.CompilerServices"u8, "Continuation"u8);
```